modules/specifying-oc-log-levels.adoc

```diff
@@ -18,8 +18,15 @@ You can investigate OpenShift CLI (`oc`) issues by increasing the command's log
 +
 [source,terminal]
 ----
-$ oc <options> --loglevel <log_level>
+$ oc <command> --loglevel <log_level>
```

Comment:
@mburke5678 above looks fine, but i do not think loglevel can be some thing as you  defined below. Loglevel's are already defined in the doc in the table. If command could be modified as above that should be sufficient and no other change required. thanks !!